test/e2e/12_sync.bats

```diff
@@ -48,6 +47,7 @@ function setup() {
 
 function teardown() {
   kill "$git_port_forward_pid"
+  unset GIT_SSH_COMMAND
```

Comment:
just FYI (sorry I missed it before), I don't think this is necessary because each test is run in its own process, see https://github.com/bats-core/bats-core#writing-tests

Reply:
Was aware of this but still added it for some reason :-S. Will remove it in an upcoming PR.

Reply:
Sure, no worries!!